poplar.mk

```diff
@@ -120,8 +120,8 @@ optee-os-clean: optee-os-clean-common
 l-loader: arm-tf
 	cp $(BL1_BIN) $(LLOADER_PATH)/atf
 	cp $(FIP_BIN) $(LLOADER_PATH)/atf
-	$(MAKE) -C $(LLOADER_PATH) CROSS_COMPILE="$(AARCH32_CROSS_COMPILE)"
-
```

Comment:
Why remove this blank line? Keep it please.

Reply:
this is usually a kind of rhetorical question, as I have no idea why I did that :)